### ISSUE TITLE
Z Seam Alignment - Random - Some layers start on unsupported material in empty space

### ISSUE THREAD

**Issue by nathanshearer** [Ultimaker/Cura#8452]
**Application version**
Cura 4.7.0

**Platform**
Gentoo Linux using the official AppImage

**Printer**
Creality Ender 3

**Reproduction steps**
1. Select a model with bridges
2. Set your layer height to 0.2 or smaller. 0.1 demonstrates this problem nicely.
3. Set "Z Seam Alignment" to Random
4. Watch the print when it is on the 2nd or first few layers of a bridge. This does not happen on the first bridge layer.
5. Sometimes a bridge does not join well for the firest few layers. This problem is a huge issue when the g-code attempts to start a line over an unstable bridge.

**Actual results**
Sometimes the random start postition is over a very thin bridge, which may not be complete or capable of supporting the beginning of a line.

**Expected results**
I would expect that the random start positions would prefer well supported parts of the model only and never start on an overhang until there is sufficient material to support a random start position.

**Project file**
Any model with a large bridge can demonstrate this problem. With a thin layer height, it can take several layers to get a stable bridge. However if a line starts over a bridge, their can be compounding problems for all future layers.

**Comment by Ghostkeeper** [comment]
Duplicate of https://github.com/Ultimaker/Cura/issues/5892.
